### PR TITLE
[READY-TO-MERGE] Document spaceTemplate and mySpaceTemplates queries

### PR DESCRIPTION
**Pull request by a-kent**
@andrewegan When you have time, can you do an initial review please?

### CODE REVIEW

**Review by andrewegan — APPROVED** [review]

**Comment by a-kent** [comment]
@jarusso @miguelestrada Ready for your review.

**Comment by jarusso** [comment]
where is space_template_main?

File: guides/V1_get_my_space_templates.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'my-space-templates'
+is: 'experimental'
+---
+
+# Get My Space Templates
+
+Using the _mySpaceTemplates_ query, a user can fetch a collection of the custom space templates that they have created.
```
Comment (miguelestrada) [inline guides/V1_get_my_space_templates.md]: >the custom space templates

Do we need the word "custom" here?  Not sure we have some that are custom and some that are not and distinguish them in queries...

File: guides/V1_get_space_template.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      defaultValue
+      acceptableValues {
+        id
+        displayName
+      }
+    }
+  }
+}
+~~~~
+
```
Comment (miguelestrada) [inline guides/V1_get_space_template.md:68]: Would it make sense to have a sample output?

**Review by miguelestrada — CHANGES_REQUESTED** [review]

**Comment by a-kent** [comment]
@jarusso In reply to https://github.com/watsonwork/watsonwork-developer-docs/pull/58#issuecomment-376189742, [V1_space_template_main.md](https://github.com/watsonwork/watsonwork-developer-docs/pull/52/files#diff-8a6cfdc320f8e4253de94fec08c8dca8) is in https://github.com/watsonwork/watsonwork-developer-docs/pull/52, so makes sense to finalize that PR before merging this one.

File: guides/V1_get_my_space_templates.md (hunk 1 above)
Comment (a-kent) [reply guides/V1_get_my_space_templates.md]: Removed.

File: guides/V1_get_space_template.md (hunk 2 above)
Comment (a-kent) [reply guides/V1_get_space_template.md:68]: Added.

Comment (andrewegan) [reply guides/V1_get_space_template.md:68]: I think we should refer to the explorer in this case. The explorer is more useful than the documentation for specific examples. Outputs should be a 1:1 mapping to the request fields and it is more costly to maintain the type structures in documentation also, whereas the explorer will always be up to date.

Comment (a-kent) [reply guides/V1_get_space_template.md:68]: Makes sense to me. I'm open to whatever is the consensus view here.

Edit: Well, I say consensus, but whatever is required to merge! :laughing:

Comment (a-kent) [reply guides/V1_get_space_template.md:68]: Replaced sample output from `spaceTemplate` with link to explorer.

**Review by miguelestrada — APPROVED** [review]

**Comment by miguelestrada** [comment]
52 is already IN.  @a-kent ok to merge this now?  @jarusso FYI

**Pull request merged**